configs/metric_learning/cub_resnet50_jpg.py

```diff
@@ -81,3 +94,6 @@
 
```

Comment:
metric leanring should use rn50 pretrained models and lr should be smaller such as 0.01 or 0.001, use cos lr policy may be better